src/ddmd/func.d

```diff
@@ -5294,9 +5294,18 @@ extern (C++) final class InvariantDeclaration : FuncDeclaration
  */
 extern (C++) static Identifier unitTestId(Loc loc)
 {
+    // there's a counter per module in case there are two unit tests on one line
+    static __gshared int[const(char)*] counterPerModule;
```

Comment:
@klickverbot That's a user case which is highly unlikely to ever happen in practice, but I see your point.